### ISSUE TITLE
How to install GDShare without Geode?

### ISSUE THREAD

**Issue by Alltroo** [HJfod/GDShare#32]
Dear author, how to install GDShare without Geode?

**Comment by FireDasher** [comment]
You can't. Just get [geode](https://geode-sdk.org/).

**Comment by ryder7223** [comment]
> You can't. Just get [geode](https://geode-sdk.org/).

Actually, you can :D
All you have to do is get the DLL from the .geode file by renaming the .geode extension to a .zip and extracting it, you can then use any normal DLL injector to put it into the game while it's running, I use Xenos for this, this should work for most mods. If you want to patch the mod into the binary to make it permanent, I can't help you because I don't know how to do it, but I know it is definitely possible.

**Comment by FireDasher** [comment]
Yeah but then you would have to do that every time, you should just use Geode because it's super cool.
